Insight.Database/Reliable/ReliableCommand.cs

```diff
@@ -106,13 +110,14 @@ internal Task<IDataReader> GetReaderAsync(CommandBehavior commandBehavior, Cance
 #endif
 
 #if !NODBASYNC
-		/// <inheritdoc/>
+        /// <inheritdoc/>
 		protected override Task<DbDataReader> ExecuteDbDataReaderAsync(CommandBehavior behavior, System.Threading.CancellationToken cancellationToken)
 		{
 			return ExecuteWithRetryAsync(
 				() =>
 				{
 					FixupParameters();
+                    InnerConnection.AutoOpen();

```

Comment:
Since we're in async land, this should probably be an openasync of some sort.
That being said, it's only going to hit this point on a retry, so it's not a huge issue.

But if you're trying for high-throughput, we should have an EnsureIsOpenAsync and then chain the methods here.